command/format/diff_test.go

```diff
@@ -225,7 +234,11 @@ new line
       ~ id         = "i-02ae66f368e8518a9" -> (known after apply)
       ~ more_lines = <<~EOT
             original
-          + new line
+          - long
+          + extremely long
+            multi-line
+            string
+            field
```

Comment:
This test change is to verify that multi-line strings are _not_ affected by the concise renderer, which is intentional.